src/web_server_config.cpp

```diff
@@ -240,6 +240,8 @@ bool web_server_config_deserialise(DynamicJsonDocument &doc, bool factory)
 
     StaticJsonDocument<128> event;
     event["config_version"] = config_version;
+    event["config_updates"] = doc;
```

Comment:
not sure that's intended. Should be removed here
you just need to use "config_version" to track config changes

Reply:
So it was intended at the beginning because I didn't find a way to send the full config each time. But I found it and not use that anymore. I can remove it yes ;-)